reposilite-backend/src/main/kotlin/com/reposilite/auth/Authenticator.kt

```diff
@@ -27,6 +27,8 @@ interface Authenticator {
 
     fun enabled(): Boolean
 
+    fun priority(): Int = 0
```

Comment:
I think it's better to have a priority as a floating point number - we don't have a use-case for it yet, but it's a bit more flexible by design, because it might be helpful if users would need to register something between 2 authenticators. 

I also think we should avoid a default value here - I'd put `0.0` for a default authenticator & `-1.0` for ldap.